src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -129,6 +130,9 @@
 	}
 
 	let selectedVariations: Record<string, string> = {};
+	$: if (data.product.hasVariations && data.product.variationLabels) {
+		customAmount = productPriceWithVariations(data.product, selectedVariations);
+	}
```

Comment:
```suggestion
	$: if (data.product.hasVariations) {
		customAmount = productPriceWithVariations(data.product, selectedVariations);
	}
```

test should be consistent here & in the html code below